cmake/OpenCVCompilerOptions.cmake

```diff
@@ -529,3 +529,8 @@ if(CMAKE_GENERATOR MATCHES "Visual Studio" AND CMAKE_CXX_COMPILER_ID MATCHES "MS
     endif()
   endif()
 endif()
+
+if(OPENCV_DUMP_PREPROCESSOR)
+  include(cmake/OpenCVDumpPreprocessor.cmake)
+  dump_preprocessor()
```

Comment:
> dump_preprocessor()

We don't need dedicated call (make it as a part of that file).

Reply:
Done.